apps/dh/api-dh/source/DataHub.WebApi/Program.cs

```diff
@@ -85,6 +88,9 @@
 var apiClientSettings = configuration.GetSection("ApiClientSettings").Get<ApiClientSettings>() ?? new ApiClientSettings();
 services.AddDomainClients(apiClientSettings);
 
+services.AddFeatureManagement();
+services.AddProcessManager(configuration);
```

Comment:
nitpick: is there a reason this call is placed here and not within the `AddDomainClients` extension method?

Reply:
1: The call to `AddFeatureManagement` should be registered independent of any other code, as we should support feature flags ins any .NET code, not just clients. So I believe it should be registered "in the root".
2: With regards to `AddProcessManager` we can move it. I kept it separate from `AddDomainClients` because I have to follow another pattern and wasn't sure if other developers would wan't me to mix the two coding patterns/styles.